src/heat-map/heat-map.component.ts

```diff
@@ -161,16 +162,18 @@ export class HeatMapComponent extends BaseChartComponent {
   }
 
   getXScale(): any {
+    const innerPadding = typeof this.innerPadding === 'number' ? this.innerPadding : this.innerPadding[0];
     return d3.scaleBand()
       .rangeRound([0, this.dims.width])
-      .paddingInner(0.1)
+      .paddingInner(innerPadding / this.dims.width)
```

Comment:
This calculation is incorrect. InnerPadding needs to be divided by the width of a single rectangle, not the whole chart.

The correct one would be 
`innerPadding / (this.dims.width / this.xDomain.length)` 
or normalized:
`innerPadding * this.xDomain.length / this.dims.width`